src/webview/ui/spectrogramComponent.ts

```diff
@@ -57,20 +57,61 @@ export default class WaveFormComponent {
         const scale = (maxFreq - minFreq) / height;
 
         axisContext.font = `20px Arial`;
-        for (let i = 0; i < 10; i++) {
-            axisContext.fillStyle = "rgb(245,130,32)";
-            const x = Math.round(i * width / 10);
-            const t = i * (settings.maxTime - settings.minTime) / 10 + settings.minTime;
-            if (i !== 0) axisContext.fillText(`${(t).toFixed(2)}`, x, 18);
 
-            const freq = minFreq + i * (maxFreq - minFreq) / 10;
+        // draw horizontal axis
+        if (settings.roundTimeAxis) {
+            const [nice_t, digit]: [number, number] = AnalyzeService.roundToNearestNiceNumber((settings.maxTime - settings.minTime) / 10);
+            const x_by_t = width / (settings.maxTime - settings.minTime);
+            let t = settings.minTime;
+            let loop_cnt = 0;   // safe guard
+            do {
+                t = Math.round(t / nice_t) * nice_t;
+                let x = (t - settings.minTime) * x_by_t;
+
+                axisContext.fillStyle = "rgb(245,130,32)";
+                if (width * (5 / 100) < x  && x < width * (95 / 100)) axisContext.fillText(`${(t).toFixed(digit)}`, x, 18);     // don't draw near the edge
+
+                axisContext.fillStyle = "rgb(180,120,20)";
+                for (let j = 0; j < height; j++) axisContext.fillRect(x, j, 1, 1);
+
+                t += nice_t
+            } while (t < settings.maxTime && loop_cnt++ < 100);
+        } else {        
+            for (let i = 0; i < 10; i++) {
+                axisContext.fillStyle = "rgb(245,130,32)";
+                const x = Math.round(i * width / 10);
+                const t = i * (settings.maxTime - settings.minTime) / 10 + settings.minTime;
+                if (i !== 0) axisContext.fillText(`${(t).toFixed(2)}`, x, 18); // skip first label
+
+                axisContext.fillStyle = "rgb(180,120,20)";
+                for (let j = 0; j < height; j++) axisContext.fillRect(x, j, 1, 1);
+            }
+        }
+
+        // draw vertical axis
+        const num_axes = Math.round(10 * settings.spectrogramVerticalScale);
```

Comment:
num_axes etc. are in snake_case, but other parts of the project are in lowerCamelcase, so could you please fix them?